swap_meet/decor.py

```diff
@@ -1,2 +1,28 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, id = None, condition = 0, width = 0, length = 0):
+        super().__init__(id, condition)
+        self.width = width
+        self.length = length
+
+#inherit get_category function from superclass Item, return class name
+    def get_category(self):
```

Comment:
Same comment as above. No need to redefine a child method if it would just use the inherited parent method.